### ISSUE TITLE
[BUG] Traefik v3 breaks calibre-web UI

### ISSUE THREAD

**Issue by zaggash** [linuxserver/docker-calibre-web#294]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

All JS Jquery are not working.

The most obvious issues are the dropdown menu which are not working.
Or the "+" menu in the admin area are not opening.

 - I paid attention to https://doc.traefik.io/traefik/master/migration/v2-to-v3/
 - I tried to add back this option with this middleware: https://doc.traefik.io/traefik/master/migration/v2-to-v3/#content-type-auto-detection
It didn't change anything.

If anyone has an idea !

### Expected Behavior

The dropdown menu should open a show options
The "+" menus in the admin area should expand

### Steps To Reproduce

Add traefik v3 in front of calibre web.

### Environment

```markdown
- OS: Unraid
- How docker service was installed: docker compose with traefik as reverse.
```

### CPU architecture

x86-64

### Docker creation

```bash
https://github.com/zaggash/homelab/blob/main/unraid-homelab/media/calibre-web-compose.yaml
```

### Container logs

```bash
No log in the container.
```

**Comment by j0nnymoe** [comment]
Hey @zaggash long time no speak 😊.
You need to report this to calibre web or it's a problem with your traefik config, if it's working fine without traefik, it's not something we need to change in the container.

**Comment by zaggash** [comment]
Hello  :)
Yup I agree, opened here in case it happens to someone.
I'll cross post upstream.

**Comment by zaggash** [comment]
My bad after further testing, it seems to be something in my config.
I'll dig further and report here the wrong setup

**Comment by zaggash** [comment]
For some reason the compression is causing issues in this version:
```
middlewares:
    compress-all:
      compress:
        excludedContentTypes:
          - text/event-stream
```

Closing !

**Comment by zaggash** [comment]
For tracking purpose
Maybe https://github.com/traefik/traefik/issues/9734 ?

**Comment by danepowell** [comment]
I don't think https://github.com/traefik/traefik/issues/9734 is the whole story, since I'm on v3.0.2 (supposedly fixed) and still seeing the issue.

I first noticed the menu dropdown not working on mobile after upgrading to Traefik v3 and traced it back to all the JS files served by calibre-web being just 1 byte (a semicolon). 

https://github.com/linuxserver/docker-calibre-web/issues/294#issuecomment-2099452871 didn't fix it for me. I had to disable the compress middleware altogether.
